qt/scientific_interfaces/test/ISISReflectometry/RunsTable/RunsTablePresenterProgressBarTest.h

```diff
@@ -0,0 +1,58 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
```

Comment:
```suggestion
// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
```

Reply:
@gemmaguest Could you apply this, we should use the current year for new files.

Reply:
Sure, sorry, thought I had